bdwgc/plan.sh

```diff
@@ -1,19 +1,41 @@
 pkg_name=bdwgc
 pkg_origin=core
-pkg_version=7.4.4
+pkg_version=7.6.0
 pkg_description="A garbage collector for C and C++"
 pkg_upstream_url="http://www.hboehm.info/gc/"
 pkg_license=('X11 style license')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source="http://www.hboehm.info/gc/gc_source/gc-${pkg_version}.tar.gz"
-pkg_shasum=e5ca9b628b765076b6ab26f882af3a1a29cde786341e08b9f366604f74e4db84
-pkg_deps=(core/glibc core/libatomic_ops)
-pkg_build_deps=(core/gcc core/make core/diffutils core/pkg-config)
-pkg_bin_dirs=(bin)
+pkg_dirname="gc-${pkg_version}"
+pkg_shasum="a14a28b1129be90e55cd6f71127ffc5594e1091d5d54131528c24cd0c03b7d90"
+pkg_deps=(
+  core/glibc
+)
+pkg_build_deps=(
+  core/diffutils
+  core/file
+  core/gcc
+  core/libatomic_ops
+  core/make
+  core/pkg-config
+)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
-pkg_dirname="gc-${pkg_version}"
+pkg_pconfig_dirs=(lib/pkgconfig)
+
+do_prepare() {
+  if [[ ! -r /usr/bin/file ]]; then
```

Comment:
Not a blocker, but it might make more sense to use -x (can execute /usr/bin/file), or -e (/usr/bin/file exists)

Reply:
This is actually what's done in other packages :) if you think we should change it, might be worth changing it in them too?